Documentation/BasicUsage.md

```diff
@@ -0,0 +1,267 @@
+# Basic Usage
+
+For starters we're going to import everything ready to use.  
```

Comment:
Since this is a "basic" usage section, how about we encourage using `registerGlobals` ?

Reply:
Will be coming with a general description in the next commit.